task/08-objects-tasks.js

```diff
@@ -23,7 +23,11 @@
  *    console.log(r.getArea());   // => 200
  */
 function Rectangle(width, height) {
-    throw new Error('Not implemented');
+  this.width = width
+  this.height = height
+  this.__proto__.getArea = function() {
+    return this.width * this.height
+  }
```

Comment:
Please,   move this logic outside Rectangle function, because everytime function Rectange be invoked - every time prototype will be changed

But be aware about context (this)